src/RPCClient.js

```diff
@@ -29,7 +32,11 @@ async function request(url, method, params, options = {}) {
   }
   const { data } = res;
   if (data.error) {
-    throw new Error(`DAPI RPC error: ${method}: ${data.error.message}`);
+    if (data.error.data) {
+      throw new DriveError(`DAPI RPC error: ${method}: ${data.error.message}: ${JSON.stringify(data.error.data)}`);
+    } else {
+      throw new CoreError(`DAPI RPC error: ${method}: ${data.error.message}`);
```

Comment:
Drive error might be without `data` too. The `data` field is part of JSON RPC specification so I would not assign error for specific services. Let's have just one error `RPCError`.

Reply:
Also, I would not set `data` into the message. It's better to pass it into error class constructor and provide a getter.

Reply:
updated